setup.py

```diff
@@ -29,7 +29,7 @@ def readme():
 	  author = 'Nikhil Kumar Singh',
 	  author_email = 'nikhilksingh97@gmail.com',
 	  license = 'MIT',
-	  packages = ['clix'],
+	  packages = ['clix',''],
```

Comment:
I don't find it anywhere , i read the official documentation on packaging , and tried this.
I can also do it other way , including icon.png in clix folder and specifying it in MANIFEST.IN